src/ds/advanced_mode/presets.cpp

```diff
@@ -361,7 +361,41 @@ namespace librealsense
         p.depth_gain.gain = 16.f;
     }
 
-    void default_430(preset& p)
+    void default_430_color_only(preset& p)
```

Comment:
Names are misleading. Do you want to set RGB sensor separately from Depth sensor? Call is `default_xxx_rgb` and `default_xxx_depth`.
Notice `p.color_control` and `p.cc` are actually depth sensor settings, see https://github.com/IntelRealSense/librealsense/blob/development/src/ds/advanced_mode/advanced_mode.cpp#L787